saleor/checkout/complete_checkout.py

```diff
@@ -130,6 +130,7 @@ def _process_voucher_data_for_order(checkout_info: "CheckoutInfo") -> dict:
         deactivate_voucher_code(code)
     return {
         "voucher": voucher,
+        "voucher_code": code.code,
```

Comment:
Nitpick: it's a bit wierd to have `code.code`, maybe we could change `code` to `voucher_code` or something like that to make it easier to read?